pyproject.toml

```diff
@@ -0,0 +1,177 @@
+[build-system]
+requires = [
+    "setuptools>=61.2",
+    "setuptools_scm[toml]>=7"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools]
+include-package-data = true
+zip-safe = false
+license-files = [
+    "LICENSE.md",
+]
+py-modules = []
+
+[tool.setuptools.package-dir]
+"" = "src"
+
+[tool.setuptools.packages.find]
+where = ["src"]
+
+[tool.setuptools_scm]
+write_to = "src/legenddataflowscripts/_version.py"
+
+[project]
+name = "legend_dataflow_scripts"
```

Comment:
let's think about a shorter name without underscores?

Reply:
i've switched to dashes